src/webpack/index.ts

```diff
@@ -75,3 +84,55 @@ function removeMiniCssExtract(config: any) {
     }
   });
 }
+
+function getLibraryName(options: any) {
+  if (options?.customWebpackConfig?.libraryName) {
+    return options.customWebpackConfig.libraryName;
+  }
+
+  const projectName = getProjectNameFromAngularJson();
+  if (projectName) return projectName;
+
+  console.warn(
+    'Warning: single-spa-angular could not determine a library name to use and has used a default value.',
+  );
+  console.info('This may cause issues if this app uses code-splitting or lazy loading.');
```

Comment:
Using .info() for the follow up lines instead of .warn(), because multiple lines of bold yellow only makes things hard to read.